src/test/java/networkbook/logic/commands/CommandResultTest.java

```diff
@@ -12,27 +10,27 @@ public class CommandResultTest {
     public void equals() {
         CommandResult commandResult = new CommandResult("feedback");
 
-        // same values -> returns true
-        assertTrue(commandResult.equals(new CommandResult("feedback")));
-        assertTrue(commandResult.equals(new CommandResult("feedback", false, false)));
+        // same values -> equal
+        assertEquals(commandResult, new CommandResult("feedback"));
+        assertEquals(commandResult, new CommandResult("feedback", false, false));
 
-        // same object -> returns true
-        assertTrue(commandResult.equals(commandResult));
+        // same object -> equal
+        assertEquals(commandResult, commandResult);
 
-        // null -> returns false
-        assertFalse(commandResult.equals(null));
+        // null -> not equal
+        assertNotEquals(commandResult, null);
 
-        // different types -> returns false
-        assertFalse(commandResult.equals(0.5f));
+        // different types -> not equal
+        assertNotEquals(commandResult, 0.5f);
 
-        // different feedbackToUser value -> returns false
-        assertFalse(commandResult.equals(new CommandResult("different")));
+        // different feedbackToUser value -> not equal
+        assertNotEquals(commandResult, new CommandResult("different"));
 
-        // different showHelp value -> returns false
-        assertFalse(commandResult.equals(new CommandResult("feedback", true, false)));
+        // different showHelp value -> not equal
+        assertNotEquals(commandResult, new CommandResult("feedback", true, false));
 
-        // different exit value -> returns false
-        assertFalse(commandResult.equals(new CommandResult("feedback", false, true)));
+        // different exit value -> not equal
+        assertNotEquals(commandResult, new CommandResult("feedback", false, true));
```

Comment:
I'm unsure if changing the syntax here is good, 
retaining the old syntax highlights that the method `equals` is being tested.

Reply:
I changed the syntax here because IDE gives a warning 🤣 
Also, it seems that the community standard is to use `assertEquals` in this case, because it displays the difference better, and is null-safe. Reference [here](https://stackoverflow.com/questions/41974209/what-is-the-actual-difference-between-assertequals-vs-asserttrue-in-testng#:~:text=Also%2C%20assertEquals(a%2C%20b,equals(b))%20is%20not.).

Reply:
Yes, in the general case, it is better to use `assertEquals`, but here we are testing the `equals` method, not comparing the two values, so it is better to call the method explicitly.

Reply:
Okay I've changed the test cases, please review. Thanks for your suggestion!